lib/resque/failure/sentry.rb

```diff
@@ -16,26 +17,26 @@ module Failure
     #   Resque::Failure.backend = Resque::Failure::Multiple
     #
     class Sentry < Base
-
-      def self.logger
-        @logger
+      class << self
+        attr_reader :logger
       end
 
-      def self.logger=(value)
-        @logger = value
+      class << self
+        attr_writer :logger
       end
 
       def save
         options = {}
         options[:logger] = self.class.logger if self.class.logger
```

Comment:
Not sure if we still need to keep `options[:logger]` cos it seems like it has been removed from support https://github.com/getsentry/sentry-ruby/pull/1021